openkfw/TruBudget
# 1843 Eliminate workflow dialog window stretching

### Checklist




### Description
- changes implemented so when user enters values for allocated/paid budget within create/edit workflow item dialog window, the window does not change its size (width)

### How to test

1. login as any user
2. navigate to whatever sub project and create/edit some workflow item
3. select allocated or paid checkbox and enter the budget for it
4. see that the dialog window is no longer stretching its width

Closes #1843
